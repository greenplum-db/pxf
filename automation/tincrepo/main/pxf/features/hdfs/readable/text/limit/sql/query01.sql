-- @description query01 for PXF HDFS Readable test limit

SELECT COUNT(*) FROM (SELECT * FROM text_limit LIMIT 1000) AS a;

SELECT COUNT(*) FROM (SELECT * FROM text_limit) AS b;
