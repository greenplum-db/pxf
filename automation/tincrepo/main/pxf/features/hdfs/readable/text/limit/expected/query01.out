-- @description query01 for PXF HDFS Readable test limit
SELECT COUNT(*) FROM (SELECT * FROM text_limit LIMIT 1000) AS a;
 count 
-------
  1000
(1 row)

SELECT COUNT(*) FROM (SELECT * FROM text_limit) AS b;
 count 
-------
 10000
(1 row)

